<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="browse_btn">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename_text">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="data_indicator">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>DATA UNAVALIABLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="chan_selector">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>100</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--Select Channels--</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="rank_selector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>140</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="rank_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>NNMF Rank</string>
   </property>
  </widget>
  <widget class="QPushButton" name="init_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>100</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="iter_selector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="iter_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Iteration limit</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>265</y>
     <width>335</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Idle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="thr_selector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>200</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>0.000001000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
   <property name="value">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="thr_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Error thresshold</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="thr_ticker">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>105</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cummulative</string>
   </property>
  </widget>
  <widget class="QPushButton" name="display_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>140</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="size_selector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="size_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Data size limit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save H W</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
